dashboard/app/models/script.rb

```diff
@@ -919,8 +919,8 @@ def self.add_script(options, raw_stages, new_suffix: nil, editor_experiment: nil
       assessment = nil
       named_level = nil
       bonus = nil
-      stage_flex_category = nil
-      stage_lockable = nil
+      flex_category = nil
+      lockable = nil
```

Comment:
I think it might be good to call this lesson_lockable because there are variables floating around here for lesson and levels and it will just make it clearer

Reply:
I don't think its a huge deal tho so if you disagree thats totally fine.

Reply:
I didn't care a lot about this since neither seems clearly better to me but... I'll leave it as if that's okay with you 😅